ee/tabby-db/src/job_runs.rs

```diff
@@ -71,7 +72,7 @@ impl DbConn {
         } else {
             None
         };
-        let query = Self::make_pagination_query_with_condition(
+        let query = make_pagination_query_with_condition(
```

Comment:
any reason this cannot be covered by macro?

Reply:
I didn't convert every use case which could be covered using this macro, just the one for demonstration. I was going to do the rest in follow-up PRs.

Reply:
Reasoning: You must be able to use `query_as!` with a type in order to use this new macro with it, so it's not as simple as just swapping it out, we need to convert datetime and IDs, which have been split up into separate PRs so far.